sources/engine/Xenko.VirtualReality/OpenVR/OpenVR.cs

```diff
@@ -150,6 +150,27 @@ public void Update()
             }
         }
 
+        public class TrackedDevice
```

Comment:
Not a fan of having classes with the same name in different namespaces. Can cause confusion and makes it harder to find/navigate the code.

Reply:
renamed the Xenko.VirtualReality one to TrackedItem so the one in OpenVR can keep the name of its original api.